6.bulk_Morphology_Elastic_Network/2.test_model/scripts/3.visualize_regression_multi_output.r

```diff
@@ -51,42 +41,88 @@ if (!file.exists(dirname(global_prediction_trend_path))) {
     dir.create(dirname(global_prediction_trend_path), recursive = TRUE)
 }
 # plot the data
-global_prediction_trend <- (
+global_prediction_trend_scatter <- (
     ggplot(df, aes(x=actual_value, y=predicted_value, col=shuffle))
-    + geom_point()
-    + geom_smooth(method=lm, se=TRUE,)
+    + geom_point(alpha=0.5, size=0.5)
+    # add geom smooth with each line being a different color
```

Comment:
I am a bit confused what this code comment references, looks like geom_smooth is deleted.

Reply:
I needed to remove that line of comment...